Client/Frontend/Strings.swift

```diff
@@ -89,6 +89,6 @@ extension Strings {
     public static let SettingsHomePageClear = NSLocalizedString("Settings.HomePage.Clear.Button", value: "Clear", comment: "Button in settings to clear the home page.")
     public static let SetHomePageDialogTitle = NSLocalizedString("HomePage.Set.Dialog.Title", value: "Do you want to use this web page as your Homepage?", comment: "Alert dialog title when the user opens the home page for the first time.")
     public static let SetHomePageDialogMessage = NSLocalizedString("HomePage.Set.Dialog.Message", value: "You can change this at any time in the Settings", comment: "Alert dialog body when the user opens the home page for the first time.")
-    public static let SetHomePageDialogYes = NSLocalizedString("HomePage.Set.Dialog.OK", value: "Yes Please", comment: "Button accepting changes setting the home page for the first time.")
-    public static let SetHomePageDialogNo = NSLocalizedString("HomePage.Set.Dialog.Cancel", value: "No Thanks", comment: "Button cancelling changes setting the home page for the first time.")
+    public static let SetHomePageDialogYes = NSLocalizedString("HomePage.Set.Dialog.OK", value: "Set Homepage", comment: "Button accepting changes setting the home page for the first time.")
+    public static let SetHomePageDialogNo = NSLocalizedString("HomePage.Set.Dialog.Cancel", value: "Cancel", comment: "Button cancelling changes setting the home page for the first time.")
```

Comment:
String changes mean we also need to rev the IDs, so change `HomePage.Set.Dialog.OK` -> `HomePage.Set.Dialog.OK2` and `HomePage.Set.Dialog.Cancel` -> `HomePage.Set.Dialog.Cancel2`.